nixos/modules/services/hardware/fancontrol.nix

```
@@ -0,0 +1,33 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.fancontrol;
+
+in {
+
+  options.hardware.fancontrol = {
+    enable = mkEnableOption "Whether to enable fancontrol (requires a configuration file, see pwmconfig)";
```

```suggestion
    enable = mkEnableOption "fancontrol (requires a configuration file, see pwmconfig)";
```
The final result will show: "Enable fancontrol (requires a configuration file, see pwmconfig)"